client/client.go

```diff
@@ -693,58 +600,23 @@ func newDefaultRequester(client *Client, opts *Config) (*defaultRequester, error
 		}
 	}
 
-	requester.doer = &http.Client{Transport: requester.transport}
+	requester.doer = &http.Client{Transport: requester.transport,
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			if req.URL.Scheme != "http" {
+				return errors.New("only HTTP redirects are allowed in FIPS builds")
+			}
+			// Allow HTTP redirects up to 10 times (Go default)
+			if len(via) >= 10 {
+				return errors.New("stopped after 10 redirects")
+			}
+			return nil
+		}}
```

Comment:
Let's put one `http.Client` field per line, and match the braces:
```suggestion
	requester.doer = &http.Client{
		Transport: requester.transport,
		CheckRedirect: func(req *http.Request, via []*http.Request) error {
			if req.URL.Scheme != "http" {
				return errors.New("only HTTP redirects are allowed in FIPS builds")
			}
			// Allow HTTP redirects up to 10 times (Go default)
			if len(via) >= 10 {
				return errors.New("stopped after 10 redirects")
			}
			return nil
		},
	}
```